### PR TITLE
Flex classes created and the new file imported.

### PR DESCRIPTION
**Pull request by rapitkan**

### CODE REVIEW

File: src/sass/atoms/_flex.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
+$flexClasses: (
+    flex: ( display: flex ),
+
+    flex-ai-fs: ( align-items: flex-start ),
+    flex-ai-fe: ( align-items: flex-end ),
+    flex-ai-c: ( align-items: center ),
+    flex-ai-b: ( align-items: baseline ),
+    flex-ai-s: ( align-items: stretch ),
+    flex-ai-inherit: ( align-items: inherit ),
+    flex-ai-initial: ( align-items: initial ),
```
Comment (niant) [inline src/sass/atoms/_flex.scss]: How about pattern: flex-align-start, flex-align-end...?

File: src/sass/atoms/_flex.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    
+    flex-jc-fs: ( justify-content: flex-start ),
+    flex-jc-fe: ( justify-content: flex-end ),
+    flex-jc-c: ( justify-content: center ),
+    flex-jc-sa: ( justify-content: space-around ),
+    flex-jc-sb: ( justify-content: space-between ),
+    flex-jc-se: ( justify-content: space-evenly ),
+    flex-jc-initial: ( justify-content: initial ),
+    flex-jc-inherit: ( justify-content: inherit ),
```
Comment (niant) [inline src/sass/atoms/_flex.scss]: How about pattern flex-justify-start, flex-justify-end..?

File: src/sass/atoms/_flex.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    flex-wrap-0: ( flex-wrap: nowrap ),
+    flex-wrap-w: ( flex-wrap: wrap ),
```
Comment (niant) [inline src/sass/atoms/_flex.scss]: How about just flex-wrap?

File: src/sass/atoms/_flex.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    flex-wrap-wr: ( flex-wrap: wrap-reverse ),
```
Comment (niant) [inline src/sass/atoms/_flex.scss]: Like the one below, how about flex-wrap-reverse?

File: src/sass/atoms/_flex.scss [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    flex-wrap-inherit: ( flex-wrap: inherit ),
+
+    flex-d-c: ( flex-direction: column ),
+    flex-d-r: ( flex-direction: row ),
```
Comment (niant) [inline src/sass/atoms/_flex.scss]: How about just flex-column, flex-row? Breaks backwards compatibility, but that's likely to happen anyway for those classes.

File: src/sass/atoms/_flex.scss (hunk 1 above)
Comment (rapitkan) [inline src/sass/atoms/_flex.scss]: flex-align-space-between?

File: src/sass/atoms/_flex.scss (hunk 3 above)
Comment (rapitkan) [inline src/sass/atoms/_flex.scss]: Sounds good. Class flex-wrap is what I've been using before.

File: src/sass/atoms/_flex.scss (hunk 4 above)
Comment (rapitkan) [inline src/sass/atoms/_flex.scss]: Sounds good.

**Pull request merged**